download_and_create_reference_datasets/v02/mito/write_mito_mitomap_ht.py

```diff
@@ -0,0 +1,19 @@
+import hail as hl
+
+from download_and_create_reference_datasets.v02.mito.utils import load
+
+CONFIG = {
+    # The data source is a web page. So we download it manually and save them to the GCS.
+    'input_path': 'https://storage.googleapis.com/seqr-reference-data/GRCh38/mitochondrial/MITOMAP/Mitomap%20Confirmed%20Mutations%20Feb.%2004%202022.tsv',
+    'input_type': 'tsv',
+    'output_path': 'gs://seqr-reference-data/GRCh38/mitochondrial/MITOMAP/Mitomap Confirmed Mutations Feb. 04 2022.ht',
+    'annotate': {
+        'locus': lambda ht: hl.locus('chrM', hl.parse_int32(ht.Allele.first_match_in('m.([0-9]+)')[0])),
+        'alleles': lambda ht: ht.Allele.first_match_in('m.[0-9]+([ATGC]+)>([ATGC]+)'),
+        'pathogenic': lambda ht: hl.is_defined(ht['Associated Diseases'])
```

Comment:
I thought we were using a file that only had the pathogenic mitopmap variants? if so, how is this column helpful?

Reply:
The file contains the data copied and pasted from the web page. If we reformat the fields of `locus`, `alleles`, and `pathogenic` before saving the file, then this column will become much simpler.

Reply:
but I don't understand what purpose the `pathogenic` field serves at all, the file should only include pathogenic variants in the first place so what is the point of having an entry for every variant telling us something we already know?

Reply:
I see what you mean now. The values for the `pathogenic` field are always `true` in the data file. But we still need it when joining with other reference data. We use the `outer` join function of Hail. This means that the missing rows will have a value of `missing` for the field.
In the end, the field can have two different values (`true` or `missing`).

Reply:
Okay, but then couldn't you just make it a hail literal for true, instead of this expression which implies theres multiple possibilities for the value?